NotationV0/src/lib/install.ts

```diff
@@ -0,0 +1,109 @@
+import * as fs from 'fs';
+import * as os from 'os';
+import * as path from 'path';
+import * as taskLib from 'azure-pipelines-task-lib/task';
+import * as toolLib from 'azure-pipelines-tool-lib/tool';
+
+import { computeChecksum } from './crypto';
+
+export async function installFromURL(downloadURL: string, expectedChecksum: string | undefined, extractPath: string): Promise<void>{
+    // Download notation
+    const fileName = path.basename(downloadURL);
+    const downloadPath = await toolLib.downloadTool(downloadURL, fileName);
+
+    // Validate checksum
+    if (!expectedChecksum) {
+        console.log(`Checksum is not provided for ${downloadURL}, skip the validation`);
```

Comment:
IMO, checksum check should never be skipped.

Reply:
The `expectedChecksum` will never be empty here, so removed the validation.